Sources/Metadata.swift

```diff
@@ -68,3 +51,26 @@ struct MetadataBuilder: Builder {
                         nullable: self.nullable, example: self.example)
     }
 }
+
+extension DataType: ImmutableMappable {
+
+    public init(map: Map) {
```

Comment:
I don't want to leak ObjectMapper to consumers of the library - let's make this method internal.